wgsl/index.bs

```diff
@@ -9143,10 +9152,14 @@ where compatibility is defined by the following table.
   <tr>
       <td>{{GPUTextureSampleType/"depth"}}
   <tr><td>[=type/write-only storage texture=]
-      <td>{{GPUTextureView}}
-      <td>{{GPUBindGroupLayoutEntry/storageTexture}}
-      <td>{{GPUStorageTextureAccess}}
+      <td rowspan=3>{{GPUTextureView}}
+      <td rowspan=3>{{GPUBindGroupLayoutEntry/storageTexture}}
+      <td rowspan=3>{{GPUStorageTextureAccess}}
       <td>{{GPUStorageTextureAccess/"write-only"}}
+  <tr><td>[=type/read-write storage texture=]
+      <td>{{GPUStorageTextureAccess/"write-only"|"read-write"}}
+  <tr><td>[=type/read-only storage texture=]
+      <td>{{GPUStorageTextureAccess/"write-only"|"read-only"}}
```

Comment:
These links will need to be fixed up after we publish this PR.

Reply:
Pushed https://github.com/gpuweb/gpuweb/pull/4326/commits/4e8d4d021c8dba244df04cb785b17e927d184a7c to move this into the anchor database which is a little less hacky.